src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -1,36 +1,49 @@
 package com.kodilla.ecommercee.controller;
 
 import com.kodilla.ecommercee.domain.OrderDto;
+import com.kodilla.ecommercee.mapper.OrderMapper;
+import com.kodilla.ecommercee.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.Arrays;
 import java.util.List;
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 @RestController
 @RequestMapping("/v1/orders")
 public class OrderController {
+    private OrderRepository orderRepository;
+    private OrderMapper orderMapper;
+
+    @Autowired
+    public OrderController(OrderRepository orderRepository, OrderMapper orderMapper) {
+        this.orderRepository = orderRepository;
+        this.orderMapper = orderMapper;
+    }
+
```

Comment:
Wiem, że to żaden błąd i może tak zostać, ale nie powinniśmy trzymać się jednej koncepcji wstrzykiwania zależności? Bo wcześniej robiliśmy chyba przez pola  klasy ?